WebSocketTest/Programs/TcpServer.cs

```diff
@@ -9,10 +9,10 @@ namespace WebSocketTest.Programs
 {
 	static class TcpServer
 	{
-		static private readonly IPEndPoint endpoint = new IPEndPoint(IPAddress.Any, 8080);
+		private static readonly IPEndPoint endpoint = new IPEndPoint(IPAddress.Any, 8080);
 
-		static private bool _isAccepting = true;
-		static public int connectionAmount;
+		private static bool isAccepting = true;
```

Comment:
shouldnt this be `_isAccepting`?

Reply:
Not according to my editor...

Reply:
Welp, static stuff. Should be fixed when the TcpServer gets updated!